src/test/java/seedu/address/logic/commands/AddTuteeCommandTest.java

```diff
@@ -0,0 +1,242 @@
+package seedu.address.logic.commands;
```

Comment:
I like that you followed the naming convention for each test case.

Reply:
Thanks, Mingyang!